api/users/views.py

```diff
@@ -316,9 +315,8 @@ class UserNodes(JSONAPIBaseView, generics.ListAPIView, UserMixin, UserNodesFilte
     # overrides NodesFilterMixin
     def get_default_queryset(self):
         user = self.get_user()
-        if user != self.request.user:
-            return default_node_list_permission_queryset(user=self.request.user, model_cls=Node).filter(contributor__user__id=user.id)
-        return self.optimize_node_queryset(default_node_list_queryset(model_cls=Node).filter(contributor__user__id=user.id))
+        auth = get_user_auth(self.request)
+        return default_node_list_permission_queryset(auth=auth, model_cls=Node).filter(contributor__user__id=user.id)
```

Comment:
Previously, constructing querysets differently depending on whether user was requesting their own nodes versus another user's nodes was being done for _slight_ optimization purposes.  At this point, using `default_node_list_permission_queryset` to factor in VOL's for both use cases.